kerykeion/utilities.py

```
@@ -640,27 +657,70 @@ def extract_year_from_iso(iso_datetime_string: str) -> int:
-def format_ancient_iso(year: int, month: int, day: int, decimal_hour: float, utc_offset_hours: float) -> str:
-    """Format a date with potentially negative year as an ISO 8601 extended-year string.
+def format_timedelta_hhmm(td: timedelta) -> str:
+    """Render a duration as ``H:MM`` (rounded to whole minutes).
 
-    Produces strings like ``"-0500-03-21T12:00:00+01:35"`` for dates that
-    Python's ``datetime`` cannot represent (year < 1).
+    Shared by the report and LLM-context serializers so both surfaces display a
+    duration (e.g. the Sun's day length) in the same ``H:MM`` form rather than
+    diverging into ``str(timedelta)`` (``H:MM:SS``).
+    """
+    total_minutes = int(round(td.total_seconds() / 60))
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)